app/Notifications/NewComment.php

```diff
@@ -47,6 +47,48 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
```

Comment:
Should have an is_modo check check to force send (if comment is not anon).

Reply:
Added.